src/jit/lir.cpp

```diff
@@ -1040,8 +1168,14 @@ LIR::Range LIR::AsRange(GenTree* firstNode, GenTree* lastNode)
 //
 // Return Value: The newly constructed range.
 //
-LIR::Range LIR::SetTreeSeq(Compiler* compiler, GenTree* tree)
+// static
+LIR::Range LIR::SeqTree(Compiler* compiler, GenTree* tree)
 {
+    compiler->gtSetEvalOrder(tree);
```

Comment:
It was called in most cases already. There were some cases it was not. I could create two functions: OrderAndSeqTree() and SeqTree(). I wonder if most people think hard enough to know why should do one but not the other. That is, were the cases not calling gtSetEvalOrder on a newly constructed tree doing so on purpose or because they forgot, and it didn't matter?